tests/constants.ts

```diff
@@ -9,21 +22,58 @@ export const SPENDING_PASSWORD = '1234567890'
 export const APP_ID = 'com.emurgo.nightly'
 export const APP_ID_PARENT = 'com.emurgo.*'
 export const APP_PATH = join(process.cwd(), '/tests/app/Yoroi-Nightly.apk')
-export const RESTORED_WALLET = [
-    'ritual',
-    'nerve',
-    'sweet',
-    'social',
-    'level',
-    'pioneer',
-    'cream',
-    'artwork',
-    'material',
-    'three',
-    'thumb',
-    'melody',
-    'zoo',
-    'fence',
-    'east'
+export const RESTORED_WALLETS: RestoredWallet[] = [
+  {
+    checksum: 'CONL-2085',
+    name: '15-word',
+    phrase: [
+      'ritual',
+      'nerve',
+      'sweet',
+      'social',
+      'level',
+      'pioneer',
+      'cream',
+      'artwork',
+      'material',
+      'three',
+      'thumb',
+      'melody',
+      'zoo',
+      'fence',
+      'east',
+    ],
+    type: 1,
```

Comment:
just a nitpick here, but I try to avoid vague values like this. 
something like this reads better
`type: 15-word-mnemonic` 
`if (wallet.type === '15-word-mnemonic') ...`
you could use an `enum` to keep the readability while also keeping the number
```
enum Wallet type {
    SomeType;
    SomeOtherType
}

if (wallet.type === WalletType.SomeType) ...
```
or maybe something else like
`if (wallet.mnemonicLength === 15) ...`
but like I said, just a nitpick

Reply:
Good point, I will change it.